cloud/src/recycler/recycler.cpp

```diff
@@ -1222,6 +1309,10 @@ int InstanceRecycler::recycle_tablets(int64_t table_id, int64_t index_id, int64_
 
     int ret = scan_and_recycle(tablet_key_begin, tablet_key_end, std::move(recycle_func),
                                std::move(loop_done));
+    if (ret != 0) {
+        LOG(WARNING) << "failed to scan_and_recycle, instance_id=" << instance_id_;
```

Comment:
also log table id index_id partition id  and db id (if has)

Reply:
in the internal call `func`, it has printed more detail log when `ret != 0`